source/index.html.md

```diff
@@ -1,239 +1,504 @@
 ---
-title: API Reference
+title: The Namara API
 
 language_tabs: # must be one of https://git.io/vQNgJ
-  - shell
-  - ruby
+  - shell: cURL
   - python
```

Comment:
Yeah, this will need clearing up - as I understand it, Bryan wants Python fleshed out, but it seems like most of this stuff is language-agnostic? I'll need time with somebody to figure out exactly what goes where, what makes more sense in the centre column vs. right, etc.